lib/BUILD

```diff
@@ -1,8 +1,53 @@
 licenses(["notice"])
 
+package(default_visibility = ["//visibility:public"])
```

Comment:
The visibility of the individual `skylark_library` targets in this directory should be private so that users can only load them through `lib.bzl`. If you change that, does everything still work (`/...` builds and tests pass)?

Reply:
Yes; done.